realm/src/main/java/io/realm/DynamicRealmObject.java

```diff
@@ -95,10 +96,13 @@ public DynamicRealmObject(RealmObject obj) {
 
     /**
      * Returns the {@code boolean} value for a given field.
+     * If the field is nullable use {@link #isNull(String)} to check for {@code null} instead of using
```

Comment:
I fully understand that. My point is that we wish to minimise the number of concepts used in our documentation.